src/mongocxx/result/delete.cpp

```diff
@@ -32,5 +34,3 @@ std::int32_t delete_result::deleted_count() const {
 }  // namespace result
 MONGOCXX_INLINE_NAMESPACE_END
 }  // namespace mongocxx
-
```

Comment:
It isn't needed in .cpp files. The postlude exists to restore the state to as close to pristine as we can get after you have seen a header file as a client. That isn't a concern for .cpp files since clients never include them